### DISCUSSION TITLE
Terraform files in 2 depths folder

### DISCUSSION THREAD

**Discussion by vmatyus** [renovatebot/renovate#10889] [Request Help] [answered]
I tried to use Renovate with terraform, but I have troubles with 2 depths folder structure. 
Simply it does not detects the terraform files under folder level 2. I created a demo repo where I placed `.tf` files on various folders and there is an additional file at `env/test/version.tf`, but it is not detected by renovate. Here are the logs from Renovate Dashboard:
```
DEBUG: Using file match: \.tf$ for manager terraform
DEBUG: Matched 3 file(s) for manager terraform: env/main.tf, env/version.tf, main.tf
```

Here is the full config that I want to use:
```
{
  "extends":[
    "config:base",
    ":semanticCommits",
    ":semanticCommitTypeAll(fix)"
  ],
  "group": true,
  "groupName": "all",
  "prHourlyLimit": 10,
  "force": {
    "recreateClosed": true
  },
  "labels": ["renovate"],
  "rangeStrategy": "pin",
  "git-submodules": {
    "enabled": true
  }
}
```

Tried to use `"terraform": {"fileMatch": ["(^|/)env/test/.*\\.tf$"]},`, but still have no result:
```
DEBUG: Using file match: \.tf$ for manager terraform
DEBUG: Using file match: env/test/.*\.tf$ for manager terraform
DEBUG: Matched 3 file(s) for manager terraform: env/main.tf, env/version.tf, main.tf
```

Also tried to use `"includePaths": ["env/test/**"]` setting, but still not detected:
```
DEBUG: Using file match: \.tf$ for manager terraform
DEBUG: Using file match: (^|/)env/test/.*\.tf$ for manager terraform
DEBUG: Found 0 package file(s)
```

Can you please help me to figure out what would be the proper config?

**Comment by viceice** [comment, marked as answer]
By default renovate ignores test folders, you need to ignore `:ignoreModulesAndTests` preset or set a custom `ignorePaths`


https://docs.renovatebot.com/presets-default/#ignoremodulesandtests

**Reply by vmatyus** [reply]
Indeed, thank you 🙂
